apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/OpenAPIContentProvider.java

```diff
@@ -0,0 +1,292 @@
+    represents the logical identity for a resource, or as much of that
+    identity that is known. In FHIR, every resource must have a _logical ID_ which is
+    defined by the FHIR specification as:
+
+    `<code>Any combination of upper or lower case ASCII letters ('A'..'Z', and 'a'..'z', numerals ('0'..'9'),
+    '-' and '.', with a length limit of 64 characters. (This might be an integer, an un-prefixed OID, UUID
+    or any other identifier pattern that meets these constraints.)</code>`
+
+    This class contains that logical ID, and can optionally also contain a relative or absolute URL
+    representing the resource identity; the following are all valid values for IdType, and all might
+    represent the same resource:
+      - `<code>123</code> (just a resource's ID)`
+      - `<code>Patient/123</code> (a relative identity)`
+      - `<code>http://example.com/Patient/123 (an absolute identity)</code>`""";
+
+  /** Open API content short description for /Patient's resource bundle offset. */
+  public static final String PATIENT_START_INDEX_SHORT =
+      "The starting offset used for result pagination";
+  /** Open API content value for /Patient's resource bundle offset. */
+  public static final String PATIENT_START_INDEX_VALUE =
+      """
+    When fetching a _Bundle Response_ using pagination, this URL parameter represents an offset
+    (starting point) into the list of elements for the _Request_.
+    It is optional and defaults to 1 if not supplied.
+    A value 0 is not allowed and negative indices are not currently supported.
+    Example:
+       - `startIndex=100`""";
+
+  /** Open API content short description for /Patient's lastUpdated parameter. */
+  public static final String PATIENT_LAST_UPDATED_SHORT = "When the resource version last changed";
+  /** Open API content value for /Patient's lastUpdated parameter. */
+  public static final String PATIENT_LAST_UPDATED_VALUE =
+      """
+    Only satisfy the Search if the Beneficiary's `last_updated` Date falls within a specified _DateRange_.
+    A _DateRange_ can be defined by providing:
+      - both low and high date like `&_lastUpdated=gt2023-01-02&_lastUpdated=lt2023-05-01`
+      - a _low_ date value like `&_lastUpdated=gt2023-01-02`
+      - a _high_ date value like `&_lastUpdated=lt2023-05-01`""";
+
+  /** Open API content short description for /Patient's Part D contract ID to be used. */
+  public static final String PATIENT_PARTD_CONTRACT_SHORT = "Part D coverage contract identifier";
+  /**
+   * Open API content value for /Patient's Part D contract ID to be used in determining Part D
+   * events.
+   */
+  public static final String PATIENT_PARTD_CONTRACT_VALUE =
+      """
+    When searching for a Patient's Part D events information, this resource identifies
+    the Part D contract value that will be used when determining eligibility.
+    Example:
+       - `_has:Coverage.extension=ABCD`""";
```

Comment:
Maybe something like this?
```suggestion
       - `_has:Coverage.extension=<Part D Contract ID Here>`""";
       - `_has:Coverage.extension=ABCD`""";
```